plugins/roll.py

```diff
@@ -66,8 +96,10 @@ def handle_message(self, message: MessageEvent) -> FriskyResponse:
             if roll is None:
                 errors.append(expr)
                 results.append('???')
-            else:
+            elif roll.count <= 10000:
                 results.append(str(calculate_roll(roll)))
+            else:
```

Comment:
can we use an f-string here, just to be consistent?

Reply:
Fixed.